### DISCUSSION TITLE
Is there a branch for USB development?

### DISCUSSION THREAD

**Discussion by charlieboi0** [CE-Programming/toolchain#456] [Q&A] [answered]
I'm asking because I see that USB support set as a milestone and the docs say that work is being done on the ``usbdrvce`` branch, but I can't find any such branch. I'm new to CE programming so I may be missing something.

**Comment by adriweb** [comment, marked as answer]
It's already merged in the master branch / available in recent releases.
Documentation: https://ce-programming.github.io/toolchain/libraries/usbdrvce.html

**Reply by charlieboi0** [reply]
Oh. So does that mean that the toolchain already has USB support?

**Reply by adriweb** [reply]
Yes

**Reply by charlieboi0** [reply]
Ok thanks. I figured as much. I just wasn't sure because it wasn't included in the docs

**Reply by adriweb** [reply]
But it is included in the docs.... https://ce-programming.github.io/toolchain/libraries/usbdrvce.html etc.

**Reply by mateoconlechuga** [reply]
Yeah what docs are you looking at?

**Reply by charlieboi0** [reply]
I was looking at the [hardware overview](https://ce-programming.github.io/toolchain/static/hardware.html#usb)... that's why.
